ros/lib/utils.py

```diff
@@ -350,8 +350,18 @@ def system_allowed_in_ros(msg, reporter):
         # https://consoledot.pages.redhat.com/docs/dev/services/inventory.html#_updated_event
         if (
                 msg.get('type') == 'updated'
-                and msg.get('platform_metadata') is None
+                and is_platform_metadata_empty(msg)
         ):
             return is_valid_cloud_provider(cloud_provider)
         is_ros = msg["platform_metadata"].get("is_ros")
     return validate_ros_payload(is_ros, cloud_provider, operating_system)
+
+
+def is_platform_metadata_empty(msg):
```

Comment:
```suggestion
def is_platform_metadata_check_pass(msg):
```

Could you rename the method something similar ^^ as no longer empty check?